src/Microsoft.AspNet.Mvc.Razor/OptionDescriptors/RazorViewEngineOptions.cs

```diff
@@ -53,16 +53,16 @@ public TimeSpan ExpirationBeforeCheckingFilesOnDisk
             = new List<ViewLocationExpanderDescriptor>();
 
         /// <summary>
-        /// Gets or sets the <see cref="IFileSystem" /> used by <see cref="RazorViewEngine"/> to locate Razor files on
+        /// Gets or sets the <see cref="IFileProvider" /> used by <see cref="RazorViewEngine"/> to locate Razor files on
         /// disk.
         /// </summary>
         /// <remarks>
-        /// At startup, this is initialized to an instance of <see cref="PhysicalFileSystem"/> that is rooted at the
+        /// At startup, this is initialized to an instance of <see cref="PhysicalFileProvider"/> that is rooted at the
```

Comment:
It is (in a round-about way). It gets initialized via the OptionsSetup which gets called the first time you access `IOptionsAccessor<RazorViewEngineOptions>`. Its easier to say Startup instead